Makefile

```diff
@@ -0,0 +1,16 @@
+BINARY_NAME=analyticsEngine
```

Comment:
@donNewtonAlpha you should get @ray-milkey to help here with the standard `Makefile` layout - it will make it easier to integrate with Jenkins